code/modules/client/verbs/who.dm

```diff
@@ -70,19 +70,30 @@
 	var/msg
 
 	// when you are admin
-	if(holder)
+	if(holder || GLOB.deadmins[ckey])
+		var/list/full_admin_list = GLOB.admin_datums+GLOB.deadmins
 		msg = "<b>Current Admins:</b>\n"
-		for(var/client/C in GLOB.admins)
-			var/rank = "\improper [C.holder.rank]"
+		for(var/datum/admins/a_datum as() in full_admin_list)
+			a_datum = full_admin_list[a_datum]
+			if(!a_datum)
+				continue
+			var/client/C = a_datum.owner || GLOB.directory[a_datum.target]
+			if(!C)
+				continue
+			var/rank = "\improper [a_datum.rank]"
 			msg += "\t[C] is \a [rank]"
 
-			if(C.holder.fakekey)
-				msg += " <i>(as [C.holder.fakekey])</i>"
+			if(a_datum.fakekey)
+				msg += " <i>(as [a_datum.fakekey])</i>"
+			if(a_datum.deadmined)
+				msg += " <b>(as Deadmin)</b>"
```

Comment:
```suggestion
				msg += " <b>(Deadmined)</b>"
```

Reply:
@mystery3525 @EvilDragonfiend 

aykhtually TTS says otherwise

https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/b8126b46-3afe-4829-8898-69679578edb5

☝️ 🤓

Reply:
🤔